libdnf/hy-util.h

```diff
@@ -36,6 +36,8 @@ int hy_detect_arch(char **arch);
 
 int hy_split_nevra(const char *nevra, char **name, int *epoch,
                    char **version, char **release, char **arch);
+int hy_parse_module_spec(const char *spec, char **name, char **stream,
```

Comment:
Wasn't sure if to add a docstring here. I can do so, but was keying off the other functions in this file not having one.